index.html

```diff
@@ -51,9 +51,220 @@ <h2 class="hero-msg">Glad to see you!</h2>
+              <img src="./img/Portfolio3.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Tonic</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio4.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>Multi-Post Stories</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
+              <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button class="see-link">
+                <a href="#">See Project</a>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <main>
+      <div class="hero">
+        <div>
+          <h1 class="hero-name">About</h1>
```

Comment:
- [ ] Kindly wrap the `about` section using the `section` instead of the `main`. 
- Semantically, the `section` tag is a useful tool for organizing and structuring your content, improving accessibility, and optimizing your site for search engines.

Reply:
done :heavy_check_mark: